File: test_or.dart
import 'package:http/http.dart' as http;

void main() async {
  final key = 'sk-4ee8718729360497-0901c8-247704ef';
  final url = 'https://openrouter.ai/api/v1/models';
  
  try {
    var res = await http.get(
      Uri.parse(url),
      headers: {'Authorization': 'Bearer ' + key},
    ).timeout(Duration(seconds: 10));
    print('OpenRouter Status: ' + res.statusCode.toString());
  } catch(e) {
    print('OpenRouter Error: ' + e.toString());
  }
}

File: test_or.py
import requests
import json

key = 'sk-4ee8718729360497-0901c8-247704ef'
headers = {
    'Authorization': f'Bearer {key}',
    'Content-Type': 'application/json'
}
data = {
    "model": "google/gemini-1.5-flash",
    "messages": [{"role": "user", "content": "hello"}]
}
res = requests.post('https://openrouter.ai/api/v1/chat/completions', headers=headers, json=data)
print(res.status_code)
print(res.text)
